CHANGELOG.md

```diff
@@ -15,47 +17,49 @@
 - Added MediaStreamTrackAudioSourceNode
 - Fixed windows build by removing termion from dev-dependencies
 
-# Version 0.28.0 (2023-01-30)
+## Version 0.28.0 (2023-01-30)
 
 - Improved AnalyserNode (performance and correctness)
 - Fixed microphone input on Raspberry Pi
 - Fixed overriding channel config for PannerNode
 
-# Version 0.27.0 (2023-01-17)
+---
```

Comment:
Why the horizontal bar?

Reply:
Oops. Left over from a previous attempt to fix the issues.

Reply:
I have fixed the wrong commit and force pushed the changes. Hope this is ok.